src/tests/actions/comments.test.js

```diff
@@ -71,22 +75,41 @@ describe('comment actions', () => {
         .reply(201, {
           comment: {
             id: 1,
-            body: 'The body'
+            body: 'The body',
+            article: {
+              id: 2,
+              slug: 'some-slug'
+            }
           }
         });
 
       const expectedActions = [
         {
-          type: types.CREATE_COMMENT_REQUEST,
-          payload: true
+          type: "CREATE_COMMENT_REQUEST",
+          payload: true,
         },
         {
-          type: types.CREATE_COMMENT_REQUEST,
-          payload: false
+          type: "CREATE_COMMENT_REQUEST",
+          payload: false,
         },
         {
-          type: types.CREATE_COMMENT_SUCCESS,
-          payload: {id: 1, body: 'The body'}
+          type: "CREATE_COMMENT_SUCCESS",
```

Comment:
You imported the actiontypes as * types from ../../actionTypes/comment but here you are calling it directly should't it be  
```suggestion
          type: types.CREATE_COMMENT_SUCCESS,
```

Reply:
Yes it should. Thanks for pointing that out. I'll fix it ASAP.